### PR TITLE
Added new recommendations for GCP governance

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by rajarshee-harness — APPROVED** [review]

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md [hunk 1]
```diff
@@ -248,6 +248,268 @@ policies:
...
+**Permissions Required:** 
+- Dry Run:
+  - `bigquery.datasets.get`
+
+- Run Once:
+  - `bigquery.datasets.get`
+
+---
+### Recommendation: gcp-delete-unused-cloud-functions
+**Description:** Delete Cloud Functions that haven't been invoked in the last 30 days to reduce costs.
```
Comment (rajarshee-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md:337]:
```suggestion
**Description:** Delete Cloud Functions that haven't been invoked in the last 7 days to reduce costs.
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md [hunk 2]
```diff
@@ -248,6 +248,268 @@ policies:
...
+
+**Policy Used:** 
+```yaml
+policies:
+  - name: gcp-delete-unused-cloud-functions
+    resource: gcp.function
+    description: >
+      Delete Cloud Functions that haven't been invoked in the last 30 days to
```
Comment (rajarshee-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md:345]:
```suggestion
      Delete Cloud Functions that haven't been invoked in the last 7 days to
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md [hunk 3]
```diff
@@ -248,6 +248,268 @@ policies:
...
+  - `compute.addresses.list`
+
+- Run Once:
+  - `monitoring.timeSeries.list`
+  - `compute.addresses.list`
+  - `compute.addresses.delete`
+
+---
+### Recommendation: gcp-list-unused-redis-instances
+**Description:** List Redis instances with less than 5% CPU utilization in last 7 days
```
Comment (douglas-j-bothwell) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md:482]:
```suggestion
**Description:** List Redis instances with less than 5% CPU utilization over the last 7 days.
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md [hunk 4]
```diff
@@ -248,6 +248,268 @@ policies:
...
+      - type: metrics
+        name: redis.googleapis.com/stats/cpu_utilization
+        metric-key: resource.labels.instance_id
+        days: 7
+        value: 0.05
+        op: lte
+      
+```
+
+**Savings Computed:** The policy identifies a list of resources on which potential savings are calculated by summing up cost of each resource for last 30 days.
```
Comment (douglas-j-bothwell) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/GCP/gcp-recommendations.md:500]:
```suggestion
**Savings Computed:** The policy identifies a list of resources on which potential savings are calculated by summing up cost of each resource for the last 30 days.
```

**Review by douglas-j-bothwell — APPROVED** [review]
Some grammer tweaks otherwise LGTM

**Review by rajarshee-harness — APPROVED** [review]

**Pull request merged**